### ISSUE TITLE
Dual mining ETH

### ISSUE THREAD

**Issue by derrickimuncy** [trexminer/T-Rex#753]
I would like to be able to set a second nonce start for dual mining ETH on one GPU.  I set the start nonce range but I think that only distributes over devices, not connections.

**Comment by derrickimuncy** [comment]
While you are in there, I would like to use nonce start to specify the starting address for the start nonce range when dual mining the same coin:
example:  Computer 1 --start-nonce 0 start --nonce-range size 20000000.
                Computer 2 --start-nonce 20000000 --nonce-range size 40000000
                Computer 3 --start-nonce 40000000 --nonce-range-size 60000000..
each computer has 1 GPU and the first solution gives 2 coins but every other one only mines 1 because the nonce is the same.

Please advise notice of change.

**Comment by trexminer** [comment]
Setting nonce ranges was only introduced to work around a bug in some now-dead coin's protocol. There should be no reason to use it now. What are you trying to achieve?

**Comment by derrickimuncy** [comment]
Okay,

Just, for coins with a long solve time it becomes more important.  The
longer the solve time, the more important. Staggering the nonces and
coordinating your miners is key to trying to do something like mining
solo.  Otherwise you are just burning power and retracing your steps.
Anyways, that is why I came to T-rex, since I can control the nonce.
Spacing out nonces by 100’s of thousands of numbers doesn’t cut it.  I need
better control.  I was just interested in trying to control the nonevent
per connection if I wanted to try to split the range on a single rig and
improve the odds.  Regards.

**Comment by derrickimuncy** [comment]
Yes, I would need to set up a node and then regulate the staggering of the
nonces myself I suppose.
